pkg/security/certManager/general/certManager.go

```diff
@@ -40,56 +41,55 @@ func (c Config) Validate() error {
 type CertManager struct {
 	config Config
 
-	watcher                 *fsnotify.Watcher
-	doneWg                  sync.WaitGroup
-	done                    chan struct{}
+	fileWatcher             *fsnotify.Watcher
 	verifyClientCertificate tls.ClientAuthType
 	logger                  log.Logger
 
-	mutex      sync.Mutex
-	tlsKeyPair tls.Certificate
-	tlsCAPool  *x509.CertPool
+	mutex            sync.Mutex
```

Comment:
inner struct for mutex and members protected by the mutex? it's no longer clear which those are now

Reply:
fixed